pkg/activator/activator.go

```diff
@@ -33,3 +39,27 @@ var (
 		RevisionHeaderNamespace,
 	}
 )
+
+// GetNameAndNamespace If the headers aren't explicitly specified,
+// then decode the revision name and namespace from the Host header.
+func GetNameAndNamespace(name, namespace, host string) (string, string) {
+	if name == "" || namespace == "" {
```

Comment:
I'd convert this to a guard clause, then the rest can be outdented.

```suggestion
	if name != "" && namespace != "" {
	   return name, namespace
	}
```